### ISSUE TITLE
UX: Optional password requirements should be displayed earlier

### ISSUE THREAD

**Issue by mxr576** [Pronovix/password_enhancements#14]
"Add 2 more upper-cased letters" is an optional requirement for admin users, although it is only displayed properly if the provided password fulfills one of the hard requirements. Is it intentional?

![Peek 2019-07-09 14-29](https://user-images.githubusercontent.com/1755573/60888080-3b763300-a256-11e9-95c7-602fb0005ca1.gif)

**Comment by tnagyce** [comment]
Nope, it should be displayed immediately when you start typing, I'll look into it.
